larray/inout/excel.py

```diff
@@ -280,15 +280,32 @@ def _fill_slice(s, length):
         return slice(s.start if s.start is not None else 0, s.stop if s.stop is not None else length, s.step)
 
 
-    def _concrete_key(key, shape):
+    def _concrete_key(key, obj, ndim=2):
+        """
+        Parameters
+        ----------
+        key
+        obj : object
+            any object with a shape attribute
+
+        Returns
+        -------
+
```

Comment:
write something after Returns or do not add section Returns

Reply:
ok will do that on Monday